templates/login-2.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<title>Login</title>
+		<link href="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
+		<script src="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
+		<script src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+		<link type="text/css" href="static/login-2.css" rel="stylesheet">
+	</head>
+	<body>
+		<div class="wrapper fadeInDown">
+			<div id="formContent">
+				<!-- Tabs Titles -->
+				<!-- Icon -->
+				<div class="fadeIn first">
+					<img src="../static/img/user.png" id="icon" alt="User Icon"/>
+				</div>
+				<!-- Login Form -->
+				<form method="post">
+					<input type="text" id="login" class="fadeIn second" name="username" placeholder="Email">
+					<input type="password" id="password" class="fadeIn third" name="password" placeholder="password">
+					<input type="submit" class="fadeIn fourth" value="Log In">
+				</form>
+				<!-- Remind Passowrd -->
+				<div id="formFooter">
+					<a class="underlineHover" href="#">Forgot Password?</a>
+				</div>
+			</div>
+		</div>
+	</body>
+</html>
```

Comment:
This is the page for the first step, but how about the second step of entering the 2FA code? Is that also using the other theme?

Reply:
Added one template for this